### DISCUSSION TITLE
invoice date

### DISCUSSION THREAD

**Discussion by raystek** [kimai/kimai#2300] [Questions & Answers]
Is there anyway to change the invoice date and hence the due date?

**Comment by kevinpapst** [comment]
You configure the amount of due days in your invoice template (https://example.com/en/invoice/template).
<img width="439" alt="Bildschirmfoto 2021-01-27 um 19 25 36" src="https://user-images.githubusercontent.com/533162/106036298-70b2d200-60d5-11eb-9b77-d0acb2b5a212.png">

As I see now, it is probably a misleading label.

**Reply by raystek** [reply]
I understand I could change the due date by changing the Payment Terms field but what about the invoice date? Is there anyway to change it to not be the date of the invoice creation?

For example, sometimes my clients paid me on the spot but I don't get to create the invoice until a few days later. When I create the invoice, I'd like the date to be the date I was paid.

Sorry for all these questions...still a newbie

**Reply by kevinpapst** [reply]
Ok, now I understand what you want. But unfortunately that is not possible right now.
You might be able to achieve that with a custom plugin, but not out-of-the-box.

You can add it as feature request: https://github.com/kevinpapst/kimai2/issues/new/choose

**Reply by vic-t** [reply]
Unfortunately, the author did not open a feature request for this very valid issue, so I went ahead and did it myself, see https://github.com/kevinpapst/kimai2/issues/2348
